### ISSUE TITLE
Regression version 1.0.30000975 es6-module doesn't include Safari 10.1 and returns incorrect data for 9.1

### ISSUE THREAD

**Issue by alan-agius4** [ben-eb/caniuse-lite#28]
Version 1.0.30000975 introduced a regression were `safari` `10.1` is not being returned when checking `es6-module` feature other than that it is returning that `safari 9.1` has partial support which is not the case. 

This data doesn't match with the `caniuse-db` and worked correctly in the previous version of caniuse-lite.

`caniuse-db`
```
"safari": {
  "3.1": "n",
  "3.2": "n",
  "4": "n",
  "5": "n",
  "5.1": "n",
  "6": "n",
  "6.1": "n",
  "7": "n",
  "7.1": "n",
  "8": "n",
  "9": "n",
  "9.1": "n",
  "10": "n",
  "10.1": "a #4 #5",
  "11": "y",
  "11.1": "y",
  "12": "y",
  "12.1": "y",
  "13": "y",
  "TP": "y"
},
```

`caniuse-lite`
```
  safari: { 
     '4': 'n',
     '5': 'n',
     '6': 'n',
     '7': 'n',
     '8': 'n',
     '9': 'n',
     '10': 'n',
     '11': 'y',
     '12': 'y',
     '13': 'y',
     '70': 'n',
     '78': 'n',
     '12.1': 'y',
     '11.1': 'y',
     '3.5': 'y',
     '3.2': 'n',
     '5.1': 'n',
     '6.1': 'n',
     '7.1': 'n',
     '9.1': 'a #4 #5'
}
```

**Comment by march08** [comment]
Should be removed from npm as well, please, it's breaking the internet!

**Comment by alan-agius4** [comment]
Seems to have been solved by `1.0.30000976`
